src/api/addresses.js

```diff
@@ -0,0 +1,29 @@
+import Http from '../http';
+
+class Addresses {
+  static async validateAddress(addressOptions={}) {
+    const {
+      countryCode,
+      stateCode,
+      city,
+      number,
+      postalCode,
+      street,
+      unit // optional
```

Comment:
also will want to add `addressLabel`.  not sure if we care to do any validation in the sdk for param combos or leave it to the api